.travis.yml

```diff
@@ -49,6 +49,10 @@ jobs:
           env: DMD=2.078.* F=production COV=1
         - <<: *test-matrix
           env: DMD=2.078.* F=devel COV=1
+        - <<: *test-matrix
+          env: DMD=2.088.* F=production COV=1
+        - <<: *test-matrix
+          env: DMD=2.088.* F=devel COV=1
```

Comment:
Just as a comment on here: the 2.078 builds are using upstream DMD rather than `dmd-transitional`.  Is that intentional?  The change was made in commit ec6cde5324da176e90509b823e55be8dcb9384b8, but it's not clear to me that that was meant to be a switch away from `dmd-transitional` (since IIRC there is also a `dmd-transitional` release for 2.078).